lib/shinq/configuration.rb

```diff
@@ -10,19 +10,20 @@ class ConfigurationError < StandardError; end
   #   You may need to set it +false+ for jobs which take very long time to proceed.
   #   You may also need to handle performing error manually then.
   class Configuration
-    attr_accessor :require, :worker_name, :db_config, :queue_db, :default_db, :process, :graceful_kill_timeout, :queue_timeout, :daemonize, :statistics, :lifecycle, :abort_on_error
+    attr_accessor :require, :worker_name, :db_config, :queue_db, :default_db, :process, :graceful_kill_timeout, :queue_timeout, :daemonize, :statistics, :lifecycle, :abort_on_error, :sleep_sec_on_error
 
     DEFAULT = {
       require: '.',
       process: 1,
       graceful_kill_timeout: 600,
       queue_timeout: 1,
       daemonize: false,
-      abort_on_error: true
+      abort_on_error: true,
+      sleep_sec_on_error: 1,
```

Comment:
maybe default can be set to 3 seconds?

Reply:
I don't know this value is reasonable or not.
So I think that default value should be minimmum.
If necessary, the user should optionally specify values that are reasonable for the user.

Reply:
I see.
We can take that option, and I leave the value to be customized by user since this is the last resort against a queue causing consecutive errors.